MODULE.bazel

```diff
@@ -49,16 +50,10 @@ python.toolchain(
     python_version = "3.11",
 )
 
-bazel_dep(name = "googleapis", version = "0.0.0", repo_name = "com_google_googleapis")
-archive_override(
+single_version_override(
     module_name = "googleapis",
-    integrity = "sha256-qfsB7M2N7tXY0blSHgIgq8/yUc0Kskid9Zk8H3OS5nc=",
```

Comment:
It looks like we already updated `googleapis` to a more recent version than is in BCR. And generated code based on the newer protos.